src/mozanalysis/frequentist_stats/linear_models/functions.py

```diff
@@ -589,6 +701,10 @@ def compare_branches_lm(
     and `summarize_joint` for more information on the output structure.
 
     """
+
+    if alphas is None:
+        alphas = [0.01, 0.05]
+
```

Comment:
I'd suggest we just make this the default for the parameter in the function signature.

Reply:
Ignored to avoid mutable data structures as default parameter values